osbenchmark/telemetry.py

```diff
@@ -380,7 +383,7 @@ class CcrStatsRecorder:
     Collects and pushes CCR stats for the specified cluster to the metric store.
     """
 
-    def __init__(self, cluster_name, client, metrics_store, sample_interval, indices=None):
+    def __init__(self, cluster_name, client, metrics_store, sample_interval, indices=None, max_replication_lag_seconds=60*60*60):
```

Comment:
I think the default value for `max_replication_lag_seconds` can be removed here, since a default is already defined on line 347.

Reply:
Sure.